website/pages/docs/plugins/sources/googleanalytics/configuration.md

```diff
@@ -0,0 +1,127 @@
+# Google Analytics Source Plugin Configuration Reference
+
+## Example
+
+This example syncs from Google Analytics to a Postgres destination.
+The (top level) source spec section is described in the [Source Spec Reference](https://www.cloudquery.io/docs/reference/source-spec).
+
+```yaml copy
+kind: source
+# Common source-plugin configuration
+spec:
+  name: googleanalytics
+  path: cloudquery/googleanalytics
+  version: "VERSION_SOURCE_GOOGLEANALYTICS"
+  tables: ["*"]
+  destinations: ["postgresql"]
+  
+  # Google Analytics specific configuration
+  spec:
+    property_id: "<YOUR_PROPERTY_ID_HERE>"
+    oauth:
+      access_token: "<YOUR_OAUTH_ACCESS_TOKEN>"
+    reports:
+    - name: example
```

Comment:
I was able to create 2 reports with the same name:
![image](https://user-images.githubusercontent.com/26760571/223713507-40689485-1f2d-46a4-a3e9-5127571b1267.png)

Are those supposed to be unique? Am I using the wrong configuration?

Reply:
Reports created in the Google Analytics UI don't matter as we're running a manually created report request.
The names of the reports in the plugin config have to be unique.